log/log.py

```diff
@@ -10,3 +10,6 @@
 dictConfig(LOG_CONFIG)
 
 LOG = logging.getLogger('ZenysisLogger')
+
+def setup_stackdriver_logger(logger):
+    pass
```

Comment:
Should we go ahead and remove stackdriver?

Reply:
Removed